oasis3/generate_slurm_script.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#SBATCH --account=rrg-pbellec
```

Comment:
Here I see you want to run it on a computing node, you would specify the time needed and potentially the computing resource.

Right now there's nothing, so it will just fail when it hits the default wall time (I have no idea what is it). 
According to the attached output, the time was not even enough to index the dataset!

```suggestion
#SBATCH --account=rrg-pbellec
#SBATCH --job-name=oasis3fmriprepslurm
#SBATCH --output=/path/to/your/scratch/logs/%x_%A.out
#SBATCH --error=/path/to/your/scratch/logs/%x_%A.err
#SBATCH --time=6:00:00
#SBATCH --cpus-per-task=1
#SBATCH --mem=4G
```

Notes:
`%x` - job name
`%A` - job ID

You might have to experiment with the time a bit. I will guess from how many subjects are there?

Reply:
Thank you! If I try to run without `#SBATCH --account=rrg-pbellec` I get an error `You are associated with multiple _cpu allocations...`

Time limit without specifying is 1hour. There are 1377 subjects. I will try with this update :)

Reply:
> If I try to run without `#SBATCH --account=rrg-pbellec`

You always need that. The change I suggested still includes that.

Next time you can use the commit suggestion button to apply the changes to your remote, and the pull to the origin to use the latest code

Reply:
Yep just explaining why I added it to the original script.

Ah OK, thanks!

Reply:
Yeah the original script was intended to just run on a interactive session like a regular bash script.

Reply:
Ah, got it!